### DISCUSSION TITLE
¿Controlled enablePagination (toggle button) or others options of table using the ref ?

### DISCUSSION THREAD

**Discussion by jesus-osornio** [KevinVandy/material-react-table#1187] [General]
Is there a way to controlled enablePagination option of table definition using de table ref of the hook or renderBottomToolbar  .

I have custom bottomToolbar using renderBottomToolbar and i added a toggle button to enable or disable pagination.  I guess it's using the table.setOptions method but not change the pagination

**Comment by KevinVandy** [comment]
What version?
